edgemesh/pkg/common/utils_test.go

```diff
@@ -0,0 +1,8 @@
+package common
+import (
+	"testing"
+)
+
+func TestSplitServiceKey(t *testing.T) {
+	SplitServiceKey("name")
```

Comment:
Unit testing consists of at least three steps:
1. Construct test data
2. Execute the tested method
3. Verify test results

You only made 1 and 2